site/content/resources/entrepreneurship/_index.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Let's move all the resources from https://techqueria.org/resources/bay-area/ to here and add all the aliases from there here as well since they're so similar

Reply:
Let's also make sure to update the navbar to reflect this new category of resources

Reply:
Hey @fvcproductions, can you clarify if by migrating all the resources from https://techqueria.org/resources/bay-area/ to https://techqueria.org/resources/entrepreneurship/, are we removing the bay-area resource?

Reply:
we are going to be redirecting https://techqueria.org/resources/bay-area/ to https://techqueria.org/resources/entrepreneurship/ instead of removing it - that's how the alias feature works so technically yes we are but users who have it referenced should be redirected to the new page 💯 

https://gohugo.io/content-management/urls/#how-hugo-aliases-work

Reply:
I see what you mean. Sounds good. Working on it. Thanks!